assets/i18n/qqq.json

```diff
@@ -104,6 +104,8 @@
 	"I18N_CONTACT_PAGE_PARAGRAPH_9": "Text on Oppia's Contact Page - This text serves as a guide for people interested in getting involved with Oppia.",
 	"I18N_CONTACT_PAGE_PARAGRAPH_9_HEADING": "Heading for the text on Oppia's Contact page.",
 	"I18N_CONTINUE_REGISTRATION": "Text displayed on a button that, when clicked, allows the user to continue the signup process.",
+	"I18N_COOKIE_BANNER_ACKNOWLEDGE": "Text displayed on a button at the bottom of the cookie banner. It allows the user to acknowledge and dismiss the banner.",
+    "I18N_COOKIE_BANNER_EXPLANATION": "Text shown in the cookie banner at the bottom of the page.",
```

Comment:
Can you fix indentation here?

Reply:
Oh! Interesting. On my editor this looks properly aligned for some reason.. 
![image](https://user-images.githubusercontent.com/11008603/125286485-7cb7e680-e339-11eb-8e51-f3c29abe7fb3.png)

Thanks for reporting it! I will fix it in my next PR (unless this has already been fixed by then).

Reply:
Ah, looks like it is using 4 spaces instead of a tab (other keys seems to be indented using tabs). Maybe that's causing this issue?

Reply:
Fixed in https://github.com/oppia/oppia/pull/13353/commits/1612584eae972abf8e3fdbef6dec69ef749ce9b4. Thanks!